### PR TITLE
BaSP-MR-126: Moved super admins GET and DELETE logic

### PR DESCRIPTION
**Pull request by Federico-Peralta**
- Created the constants, reducer, thunks and actions for Super admins
- Added Super admins reducer to the rootReducer
- Moved GET and DELETE logic and corrected the index
- Added spinner

### CODE REVIEW

File: src/Components/SuperAdmins/index.jsx [hunk 1]
```diff
@@ -88,36 +70,44 @@ const SuperAdmins = () => {
...
+            typeStyle={'success'}
+            title={'Success'}
+            body={alertMessage}
+          />
+          <SharedModal
+            isDelete={false}
+            show={showAlert}
+            closeModal={handleExitAlert}
+            title={'Something is wrong'}
+            body={alertMessage}
```
Comment (EmaBaldor) [inline src/Components/SuperAdmins/index.jsx:104]: You can for later simplify the modals.

**Review by EmaBaldor — COMMENTED** [review]
Good job!

**Review by EmaBaldor — APPROVED** [review]

File: src/Redux/superadmins/constants.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const GET_SUPERADMINS_PENDING = 'GET_SUPERADMINS_PENDING';
+export const GET_SUPERADMINS_SUCCESS = 'GET_SUPERADMINS_SUCCESS';
+export const GET_SUPERADMINS_ERROR = 'GET_SUPERADMINS_ERROR';
+export const GET_SUPERADMINSID_PENDING = 'GET_SUPERADMINSID_PENDING';
```
Comment (lorenzomauro59) [inline src/Redux/superadmins/constants.js]: Should be 'GET_SUPERADMINS_BY_ID_PENDING' to make consistence

File: src/Redux/superadmins/thunks.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      return data;
+    } catch (error) {
+      dispatch(getSuperAdminsError(error));
+    }
+  };
+};
+
+export const getSuperAdminById = (id) => {
+  return async (dispatch) => {
+    dispatch(getSuperAdminsIdPending);
```
Comment (AleMac17) [inline src/Redux/superadmins/thunks.js]: getSuperAdminsIdPending is not with ()?

**Review by lorenzomauro59 — APPROVED** [review]
Only the comment that my partners and I made, rest is working and ok!!

**Comment by Federico-Peralta** [comment]
[[Super Admins] - Move get and delete logic](https://trello.com/c/VZMfPLEl/126-super-admins-move-get-and-delete-logic)

**Comment by nazarenapantanetti** [comment]
Tested and approved!

**Pull request merged**